src/Templates/src/templates/maui-mobile/Resources/Styles.xaml

```diff
@@ -0,0 +1,164 @@
+﻿<ResourceDictionary 
+    xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+    xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml">
+
+    <Color x:Key="AccentColor">#512bdf</Color>
+    <Color x:Key="PrimaryLightColor">White</Color>
+    <Color x:Key="PrimaryDarkColor">Black</Color>
+
+    <Color x:Key="SecondaryAccentColor">#5639b0</Color>
+    <Color x:Key="SecondaryLightColor">LightGray</Color>
+    <Color x:Key="SecondaryDarkColor">DarkSlateGray</Color>
+
+    <Style TargetType="Button">
+
+        <!-- BUTTON SPECIFIC PROPERTIES -->
+        <Setter Property="TextColor" Value="{AppThemeBinding Light={StaticResource PrimaryLightColor}, Dark={StaticResource AccentColor}}" />
+        <!--<Setter Property="CharacterSpacing" Value="0" />-->
```

Comment:
Ultimately I think we'll want to remove the commented out ones from the templates, and leave them for a comprehensive list of styles somewhere else in our repo to be referenced by docs